templates/blog.html

```diff
@@ -34,4 +39,10 @@ <h4 class="modal-title">Confirm Deletion</h4>
     </div>
     {% endif %}
 </div>
+<div class="row">
+    <a href="/blog">
+        <span class="glyphicon glyphicon-chevron-left"></span>
+        Back to {{site_name}} Blog
```

Comment:
How about adding another link back to the blog index on the top, above the post?
I think this would be convenient for long blog posts, if you misclicked on the wrong post or scrolled up after reading it or whatever. At least in my testing I missed it :grin:

Reply:
Added!